tests/integration/__init__.py

```diff
@@ -392,6 +392,9 @@ def _id_and_mark(f):
 xfail_scylla = lambda reason, *args, **kwargs: pytest.mark.xfail(SCYLLA_VERSION is not None, reason=reason, *args, **kwargs)
 incorrect_test = lambda reason='This test seems to be incorrect and should be fixed', *args, **kwargs: pytest.mark.xfail(reason=reason, *args, **kwargs)
 
+lessthanscylla55 = pytest.mark.skipif(SCYLLA_VERSION is not None and Version(get_scylla_version(SCYLLA_VERSION)) < Version('5.5'),
```

Comment:
can't we just change the test, not to assume the extensions is empty, and keep the rest of the test working checking for the existence of the custom extensions created in the test ? 

cause this condition almost like deleting the test, it soon would stop working completely

Reply:
Done